File: benchmark/comparison/results/blind_py/15.py
def reverse_list(lst):
    return lst[::-1]

def fmt(lst):
    return "[" + ", ".join(str(x) for x in lst) + "]"

print(fmt(reverse_list([1, 2, 3])))
print(fmt(reverse_list([])))
print(fmt(reverse_list([1])))

File: benchmark/comparison/results/blind_ts/15.ts
function reverse(lst: number[]): number[] {
  const result: number[] = [];
  for (let i = lst.length - 1; i >= 0; i--) result.push(lst[i]);
  return result;
}

function formatList(lst: number[]): string {
  return "[" + lst.join(", ") + "]";
}

console.log(formatList(reverse([1, 2, 3])));
console.log(formatList(reverse([])));
console.log(formatList(reverse([1])));
